listener/avalon_listener/tcs_work_order_handler_sync.py

```diff
@@ -90,7 +90,6 @@ def WorkOrderSubmit(self, **params):
                 data
             )
 
-        worker_id = input_value_json["params"]["workerId"]
         # Check if workerId is exists in avalon
```

Comment:
Move the comment down or delete the other line.

Reply:
Done